tmt/steps/provision/testcloud.py

```diff
@@ -469,10 +469,13 @@ def start(self) -> None:
 
         # Prepare ssh key
         # TODO: Maybe... some better way to do this?
-        if "coreos" in self.image.lower():
+        if re.search('coreos|rhcos', self.image.lower()):
             self._instance.coreos = True
             # prepare_ssh_key() writes key directly to COREOS_DATA
             self._instance.ssh_path = []
+        # rhcos does not allow root login
+        if "rhcos" in self.image.lower():
+            self.user = "core"
```

Comment:
I'd advise against overwriting the value user might have provided via the `--user` option. I'd argue it should be respected, something like this, maybe?

```python
if 'rhcos' in self.image.lower() and not self.opt('user'):
    self.user = 'core'
```

Reply:
Looks we can do that for also for fedora-coreos - https://docs.fedoraproject.org/en-US/fedora-coreos/authentication/#_default_user

Reply:
Updated, thanks for the review.
IMU, fedora-coreos allows `root` login.